playwright-e2e/dsm/pages/kits-page-base.ts

```diff
@@ -0,0 +1,191 @@
+import { Locator, Page, expect } from '@playwright/test';
+import DsmPageBase from './dsm-page-base';
+import { KitsTable } from 'dsm/component/tables/kits-table';
+import { KitTypeEnum } from 'dsm/component/kitType/enums/kitType-enum';
+import { waitForNoSpinner, waitForResponse } from 'utils/test-utils';
+import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import { KitType } from 'dsm/component/kitType/kitType';
+import { logInfo } from 'utils/log-utils';
+import { KitsColumnsEnum } from './kitsInfo-pages/enums/kitsColumns-enum';
+import Modal from 'dsm/component/modal';
+import { assertTableHeaders } from 'utils/assertion-helper';
+
+export default abstract class KitsPageBase extends DsmPageBase {
```

Comment:
`KitsPageBase.ts` is the new class that helps to remove duplicated code in kits pages.